src/project/ProjectManager.js

```diff
@@ -635,7 +635,8 @@ define(function (require, exports, module) {
         if (!projectRoot) {
             return;
         }
-        model.setScrollerInfo($projectTreeContainer[0].scrollWidth, $projectTreeContainer.scrollTop(), $projectTreeContainer.scrollLeft(), $projectTreeContainer.offset().top);
+        //We don't want to allow the user to scroll off the right, hiding the files
```

Comment:
@davidhuang1550 would you be able to prefix your comment text with: `XXXbramble: `? This will help us identify this later as a change that we made and not Adobe Brackets.